### DISCUSSION TITLE
Issue applying mask transform to 4d tensor

### DISCUSSION THREAD

**Discussion by cbri92** [TorchIO-project/torchio#893] [Q&A]
Hello,
I am trying to apply a brain mask (1,l,w,d) to a 4d series (DSC MRI sequence) (120,l,w,d), however, when I run the code I get an error message saying that the transform cannot be applied because there is a mismatch in the channel dimension between the label and the image. I guess this issue is somewhat related to a previous issue  "Support 4D tensors as torchio.Image #213 ", but I was wondering if there was a straightforward way to implement this transformation?

**Comment by romainVala** [comment]
Hello 
with a small reproducible example it is easier for some else to understand ...
I guess you are referring to the `ApplyMask` transform ?
for this it would make sens to broadcast the mask in the channel dimension, I guess a quick fix could be donne to the transform ...

**Comment by fepegar** [comment]
Hi, @cbri92. I suggest you open an issue for this.

**Reply by cbri92** [reply]
thanks @romainVala and @fepegar . I have now opened an issue on this
